Src/BootCamp.Chapter/Person.cs

```diff
@@ -1,7 +1,24 @@
-﻿namespace BootCamp.Chapter
+﻿using System;
+using System.Globalization;
+
+namespace BootCamp.Chapter
 {
     public class Person
     {
-        // add missing properties
+        public string Name { get; set; }
```

Comment:
The properties should have their correct data type and not just string because the source was a text file!

BirthDay & Gender should change types for sure.
If you really wanted to push the boundary, Country, and Email could change also. but not needed

Reply:
Changed the types for Birthday and Gender. 
The rest I can do when I have time